xbmc/filesystem/AndroidAppFile.h

```diff
@@ -56,6 +56,7 @@ class CFileAndroidApp : public IFile
 private:
   CURL              m_url;
   std::string       m_appname;
+  androidPackage    m_droidPackage;
   int               m_iconWidth;
```

Comment:
Yep. Exploding the struct would do, too, though. Just 3 fields, and 1 is a duplicate.